### ISSUE TITLE
implement a possibility to set order of the tests execution

### ISSUE THREAD

**Issue by sopos** [teemtee/tmt#944]
allowing tests to define the order would allow one to use some of the tests as a setup tasks

**Comment by lukaszachy** [comment]
FTR: Order is listed as core attribute

**Comment by lukaszachy** [comment]
One can use #950 to enumerate tests in order which they should be ran. 

Support of 'order' attribute is TODO.

**Comment by psss** [comment]
The core key `order` is outlined here:

 * https://tmt.readthedocs.io/en/latest/spec/core.html#order

I believe this is a duplicate of #486. Should be relatively easy to implement. Let's try for `1.10`.

**Comment by psss** [comment]
Maybe @ZelenyMartin would be interested in this?

**Comment by dkarpele** [comment]
I will work on this request.

**Comment by pcahyna** [comment]
@sopos is it a good idea? With this, you basically say that test can't run on separate SUTs, and can't run in parallel.

**Comment by sopos** [comment]
@pcahyna I agree that it may collide in certain situations. At the same time there may be some specific needs in a plan so it should be something like _"use it if you know what you are doing"_.

Actually having a test as a setup should be done in some different way, e.g. through dependencies or completely independent approach, I mean some relations between tests, e.g. before, after, ...
